app/app.go

```diff
@@ -138,9 +138,9 @@ func init() {
 	feemarkettypes.DefaultMinGasMultiplier = sdk.NewDecWithPrec(1, 1)
```

Comment:
Unable to wrap these changes into the last commit because the last commit is tagged as v0.3.2, the version the testnet is using.